### ISSUE TITLE
Registry configuration not shown in `pdm config`

### ISSUE THREAD

**Issue by GabDug** [pdm-project/pdm#1616]
- [X] I have searched the issue tracker and believe that this is not a duplicate.

## Steps to reproduce

Use the new 2.4.0 feature for multiple registries.

```
pdm config pypi.extra.url https://test.com/simple
pdm config pypi.extra.username "foo"
pdm config pypi.extra.password "password4foo"
```

The following is added to the config.toml as it should, in my home configuration:
```
[pypi.extra]
url = "https://registry.com/simple"
username = "foo"
password = "password4foo"
```

## Actual behavior

When running `pdm config`, the configuration is not visible.

## Expected behavior

The configuration should be visible, or at least give a warning that registries configuration are hidden.

## Environment Information

```bash
PDM version:
  2.4.0
```

The feature works great nevertheless, thanks for everything!

**Comment by frostming** [comment]
> The configuration should be visible, or at least give a warning that registries configuration are hidden.

SGTM, or showing a placeholder would be good, would you like to contribute?

**Comment by GabDug** [comment]
@frostming I'm gonna try to contribute 😃

**Comment by frostming** [comment]
Closed by #1622
